src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -187,12 +152,21 @@ def __call__(
         information.
 
         Args:
-            inputs (`np.ndarray` or `bytes` or `str`):
-                The inputs is either a raw waveform (`np.ndarray` of shape (n, ) of type `np.float32` or `np.float64`)
-                at the correct sampling rate (no further check will be done) or a `str` that is the filename of the
-                audio file, the file will be read at the correct sampling rate to get the waveform using *ffmpeg*. This
-                requires *ffmpeg* to be installed on the system. If *inputs* is `bytes` it is supposed to be the
+            inputs (`np.ndarray` or `bytes` or `str` or `dict`):
+                The inputs is either :
+                    - `str` that is the filename of the
+                    audio file, the file will be read at the correct sampling rate to get the waveform using *ffmpeg*.
+                    This
+                requires *ffmpeg* to be installed on the system.
+                    - `bytes` it is supposed to be the
                 content of an audio file and is interpreted by *ffmpeg* in the same way.
+                    - (`np.ndarray` of shape (n, ) of type `np.float32` or `np.float64`)
+                        Raw audio at the correct sampling rate (no further check will be done)
+                    - `dict` form can be used to pass raw audio sampled at arbirary `sampling_rate` and let
+                this pipeline do the resampling. The dict must be in the fomat `{"sampling_rate": int, "raw":
+                np.array}` with optionally a `"stride": (left: int, right: int)` than can ask the pipeline to treat the
+                first `left` samples and last `right` samples to be ignored in decoding (but used at inference to
+                provide more context to the model). Only use `stride` with CTC models.
```

Comment:
I know we don't have a good check that will tell you the doc is failing yet, but this docstring was pretty obviously badly formatted.
Please pay closer attention before merging PRs :-)

Reply:
Could something have happened in quality ?

Will pay more attention in the future though.

Reply:
I think the styling script only exacerbated a wrong syntax (it's leaving the fixed docstring alone).